### ISSUE TITLE
Biome Tweaker crash in 1.9.4 dev env

### ISSUE THREAD

**Issue by shadowfacts** [superckl/BiomeTweaker#89]
[Crash report](https://gist.github.com/shadowfacts/8bc7c350afdc1c9ba3afaff8a4b91930)

Script (based on [this](https://github.com/superckl/BiomeTweaker/wiki/Example-Scripts#creating-a-desert-world)):

``` js
asteroid = forBiomes(217)

asteroid.addToGeneration("WARM", 2000)
asteroid.addToGeneration("COOL", 2000)
asteroid.addToGeneration("DESERT", 2000)
asteroid.addToGeneration("ICY", 2000)

asteroid.removeAllSpawns("CREATURE")

Tweaker.setStage("PRE_INIT")
asteroid.set("reedsPerChunk", 20)
asteroid.set("clayPerChunk", 20);
Tweaker.setStage("FINISHED_LOAD");

all = forAllBiomes()
all.set("genWeight", 11)
all.set("isSpawnBiome", false)

asteroid.set("isSpawnBiome", true)
asteroid.set("genWeight", 50000)
```

**Comment by superckl** [comment]
I'm not really sure what's going on. Forge is supposed to deobf any mods in the mods folder, but it is failing to deobf this method call

https://github.com/superckl/BiomeTweaker/blob/1.9/src/main/java/me/superckl/biometweaker/util/BiomeHelper.java#L162

I will provide deobf versions, but for some reason the manifest isn't getting updated in the deobf jar. Stay tuned.

**Comment by shadowfacts** [comment]
Yeah, that's definitely what's happening. I also tried using FG's `deobfCompile` but that also caused the crash, same as putting it in the mods folder.

**Comment by superckl** [comment]
Fixed with commit f359523c9d9e7f972f6abbdd996bf2b4dbd53fc6

A new version has been uploaded to CurseForge along with dev jars.

http://minecraft.curseforge.com/projects/biometweaker/files/2307890
